wavefront_python_sdk/entities/histogram/histogram_granularity.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import, division, print_function
+
+
+class HistogramGranularity(object):
```

Comment:
Why do you need a class here? Why won't you just have these as module constants?